Makefile.common_app

```diff
@@ -0,0 +1,95 @@
+#*******************************************************************************
+#   Ledger SDK
+#   (c) 2022 Ledger
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#*******************************************************************************
+
+COMMON_APP_FLAGS = 0x000
+
+#####################################################################
+#                            BLUETOOTH                              #
+#####################################################################
+ifeq ($(ENABLE_BLUETOOTH), 1)
+    COMMON_APP_FLAGS = 0x200  # APPLICATION_FLAG_BOLOS_SETTINGS
+    COMMON_APP_DEFINES += HAVE_BLE BLE_COMMAND_TIMEOUT_MS=2000 HAVE_BLE_APDU
+    COMMON_SDK_SOURCE_PATH += lib_blewbxx lib_blewbxx_impl
+endif
+
+#####################################################################
+#                               DEBUG                               #
+#####################################################################
+ifeq ($(ENABLE_DEBUG), 1)
```

Comment:
Why not use the already existing [DEBUG](https://github.com/LedgerHQ/ledger-secure-sdk/blob/2fe71ea336aa3f3c3ecb248271e950807e0a175b/Makefile.defines#L99) ?
Or if you don't want to have the optimization/debug flags tied to PRINTF being enabled, you should then rename it to something more relevant.

Reply:
I switched it back to DEBUG :)